streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1500,68 +1507,57 @@ private void validateRackAwarenessConfiguration() {
         });
     }
 
+    private void overwritePropertyMap(final Map<String, Object> props, final String key, final Object unmodifiableDefaultValue, final String config) {
+        final String overwritePropertyLogMessage = "Unexpected %s config `%s` found. User setting (%s) will be ignored and the Kafka Streams default setting (%s) will be used";
```

Comment:
```suggestion
        final String overwritePropertyLogMessage = "Unexpected %s config `%s` found. User setting (%s) will be ignored and the Kafka Streams setting (%s) will be used";
```

Reply:
I would remove "default" here, because we also log this message for `bootstrap.server` which does not have a default (it would just come from `StreamsConfig`)

Reply:
I have removed the default - resolving this